kev_lambda.py

```diff
@@ -83,5 +85,43 @@ def lambda_handler(event, context):
     if len(tweets) > 0 and len(tweets) < 100:
         tweet(tweets)
         save_new_cves(old_cve_list)
+        post_to_slack(tweets)
 
     return tweets
+
+
+def post_to_slack(tweets):
+    """Posts to slack
+
+    Args:
+        tweets ([list]): Tweets to post to slack
+    """
+
+    # Set the headers for slack HTTP POST
+    headers = {"Content-Type": "application/json"}
```

Comment:
This HTTP header is not needed , if you use. `requests.post(url, json=payload)` below